Model/Country.cs

```diff
@@ -0,0 +1,36 @@
+/// <summary>
+/// Database model for countries
+/// </summary>
+public class Country
+{
+    /// <summary>
+    /// Constructor
+    /// </summary>
+    /// <param name="A2Code">Country's A2 isocode</param>
+    /// <param name="IsEUCU">Is it a member of EUCU</param>
+    public Country(string A2Code, bool IsEUCU)
+    {
+        this.A2Code = A2Code;
+        this.IsEUCU = IsEUCU;
+    }
+
+    /// <summary>
+    /// AutoIncrement-ID of the config
+    /// </summary>
+    public int Id { get; set; }
```

Comment:
Question for all tables: Where is it defined that the Ids auto-increment? Is that the default setting or is a config missing?

Reply:
it is the default setting